### PR TITLE
Fix multiple CSS issues in footer

### LINKED ISSUE #1662: Things I don't like about the footer
Screen Shot 2020-11-06 at 3 00 20 PM

1. (in the 2-col grid) I don't like that the black rectangle around each link is clickable. It should just be the white text. It's unexpected that clicking anywhere in the black emptyness suddenly opens a link.
2. There's an odd space between the words "MDN" and "Mozilla" until their little SVG icons. Looks better on Kuma. 
3. The words "these licenses" in the sentence "© 2005-2020 Mozilla and individual contributors. Content is available under these licenses." is actually a link. You don't find that out until you hover over these two words. 
4. It's weird that some links are underlined and some are not.

### PR DESCRIPTION
**Pull request by bershanskiy**
Fixes #1662.

## Before
image

## After
image

### CODE REVIEW

**Comment by peterbe** [comment]
To me, the most pressing thing is the inconsistency of some links being underlined and some not. The inconsistency makes it look sloppy and it forces the reader to have to think.

**Comment by bershanskiy** [comment]
@peterbe
> inconsistency of some links being underlined and some not.

Please consider the following prototypes:
1. Remove underlines, make in-text link grey.
    Source of inspiration: https://blog.mozilla.org/

image

2. Remove long text, replace with short link

image

**Comment by peterbe** [comment]
I still think we should/must say that line "(c) 2005-2020 ..."
In terms of style, I'll fully defer to @schalkneethling 
All I can contribute is pushing for consistency :)

File: client/src/ui/organisms/footer/index.scss [hunk 1]
```diff
@@ -43,6 +44,10 @@
       margin-right: 10px;
     }
 
+    ul {
+      padding-inline-start: 5px;
```
Comment (schalkneethling) [inline client/src/ui/organisms/footer/index.scss:48]: We work in increments of `6px` to keep spacing consistent and use SASS variables and a little math for this so here, please do:

```suggestion
      padding-inline-start: $base-spacing / 4;
```

File: client/src/ui/organisms/footer/index.scss [hunk 2]
```diff
@@ -62,6 +67,10 @@
       margin: 50px 0 30px;
       padding: 30px 0 0;
     }
+
+    a {
+      text-decoration: underline;
+    }
```
Comment (schalkneethling) [inline client/src/ui/organisms/footer/index.scss:73]: I believe it is clear that most everything here in the footer are links. For everything to have underlines would look a bit busy(not that that is what this line will do) so, let's go with nothing underlined. The only one that could be a little cryptic is the line that ends with a link to the relevant licenses. But then again, the words "these licenses" clearly suggests to me that those words are a link to the licenses.

File: client/src/ui/organisms/footer/index.scss [hunk 3]
```diff
@@ -76,7 +85,6 @@
     a {
       display: block;
       padding: 0 20px 0 0;
-      text-decoration: underline;
```
Comment (schalkneethling) [inline client/src/ui/organisms/footer/index.scss:79]: 👍

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Two small changes and we are good to go. Thank you for your contribution @bershanskiy

File: client/src/ui/organisms/footer/index.scss (hunk 2 above)
Comment (bershanskiy) [reply client/src/ui/organisms/footer/index.scss]: Done!

**Review by schalkneethling — APPROVED** [review]
r+ 🎉 ~ Thank you for your contribution @bershanskiy

**Comment by schalkneethling** [comment]
@bershanskiy For future reference. We require that all commits be signed. Please have a lot at the following link for details. Thank you

https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-commit-signature-verification

**Pull request merged**